Jellyfin.Plugin.AniDB/Providers/equals_check.cs

```diff
@@ -79,15 +79,16 @@ public static string FuzzyRegexEscape(string a)
 
             // "words"
             a = _sAtEndBoundaryRegex.Replace(a, ".?s");
+            a = a.Replace("re", "re.?", StringComparison.OrdinalIgnoreCase);
+            a = a.Replace("Gekijyouban", "Gekijouban", StringComparison.OrdinalIgnoreCase);
+            a = a.Replace("Mahoutsukai", "Mahou ?tsukai", StringComparison.OrdinalIgnoreCase);
+            a = a.Replace("to aru", "to ?aru", StringComparison.OrdinalIgnoreCase);
+            a = a.Replace("OVA", "((OVA)|(OAD))", StringComparison.OrdinalIgnoreCase);
+            a = a.Replace("OAD", "((OVA)|(OAD))", StringComparison.OrdinalIgnoreCase);
```

Comment:
shouldn't be the `re` replacement be at the end of this list here if the Word replacements should be at the top?

Reply:
As long as it doesn't interfere with anything, it doesn't matter.

Reply:
I know that, its more a question if it should be moved down now before it overlaps or later (assuming there ever is a need for a later change) :wink: